### PR TITLE
Call getUserListenCountsMonthly in client artist dashboard

### PR DESCRIPTION
**Pull request by sabrina-kiam**
### Description
This PR updates the artist dashboard on the client to call the new getUserListenCountsMonthly endpoint to get play count data over a given time frame.

### Dragons
Maybe checking the data loads in a reasonable time since we had to join on the tracks table for this.

### How Has This Been Tested?
Ran the client against staging with my local libs/sdk changes from [this PR.](https://github.com/AudiusProject/audius-protocol/pull/4147) and logged into isaac's account with play data to confirm graph loaded correctly. Play counts should total up to/near the play count number listed under the graph.

### How will this change be monitored?
n/a but open to any ideas! 

### Feature Flags ###
No feature flags

### CODE REVIEW

File: packages/common/src/services/audius-backend/AudiusBackend.ts [hunk 1]
```diff
@@ -1152,6 +1152,29 @@ export const audiusBackend = ({
...
+          normal: (libs) => libs.User.getUserListenCountsMonthly,
+          eager: DiscoveryAPI.getUserListenCountsMonthly
+        },
+        encodeHashId(currentUserId),
+        startTime,
+        endTime
+      )
+      return userListenCountsMonthly
+    } catch (e) {
+      console.error(e)
```
Comment (dylanjeffers) [inline packages/common/src/services/audius-backend/AudiusBackend.ts:1175]: Let's use the getErrorMessage util here. It essentially casts the e: any type to string and handles all the potential error types

**Review by dylanjeffers — APPROVED** [review]
Awesome!

**Review by sliptype — APPROVED** [review]
Amazing 🙌 lgtm! Also feel free to use `sk` if you want lol, doesn't really matter if we overlap

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sak-play-stats-client

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sak-play-stats-client

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sak-play-stats-client

**Pull request merged**